docs/index.md

```diff
@@ -1,28 +1,41 @@
 # Overview
 
-pytest-splunk-addon is an open-source dynamic test plugin for Splunk Apps and Add-ons
-which allows the user to test knowledge objects, CIM compatibility and index time properties.
+pytest-splunk-addon is an open-source dynamic test plugin for Splunk [Apps](https://docs.splunk.com/Splexicon:App) and [Add-ons](https://docs.splunk.com/Splexicon:Addon)
+which allows the user to test [knowledge objects](https://docs.splunk.com/Splexicon:Knowledgeobject), [CIM](https://docs.splunk.com/Splexicon:CommonInformationModel) compatibility and [index time properties](https://docs.splunk.com/Splexicon:Indexedfield).
```

Comment:
Can you verify these links are working?
I did mkdocs serve in my local and its shows just plain text

Reply:
Yeah, I verified those links and they are working as expected.